source/includes/_155-encryption.md.erb

```diff
@@ -0,0 +1,5 @@
+# Encryption
+
+The encryption API allows users with any administrator privilege to encrypt a plain text value.
```

Comment:
The encryption API allows users with any administrator privilege to get the ciphertext(encrypted text) corresponding to any plain text value. You may use this cipher text in other APIs that allow you to ....

NOTE (in the red/or blue block of text): This API is rate limited to xxx requests per minute to prevent brute force attacks that may allow attackers to guess the cipher key.